### PR TITLE
Update: adding reference for configuring artifacts in source freshness docs

### PR DESCRIPTION
**Pull request by aescay**
## Description & motivation

This adds a final section to reference the artifacts docs. This section was taken and shared from the [Cloud Generating Documentation page ](https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-generating-documentation.md).

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

**Review by patkearns10 — APPROVED** [review]
Looks good to me!
🌵

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-snapshotting-source-freshness.md [hunk 1]
```diff
@@ -45,3 +45,7 @@ It's important that your freshness jobs run frequently enough to snapshot data l
 | 1 hour | 30 mins |
 | 1 day | 12 hours |
 | 1 week | About daily |
+
+## Easy access
+
+See [Building and Configuring Artifacts](artifacts) for more information on exposing links to the latest Documentation and Source Freshness reports to your team.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-snapshotting-source-freshness.md]:
```suggestion
For more on exposing links to the latest Documentation and Source Freshness reports to your team, see [Building and configuring artifacts](artifacts).
```

**Review by runleonarun — APPROVED** [review]
I made a minor edit, but this looks good, so let's get it merged!

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-snapshotting-source-freshness.md [hunk 2]
```diff
@@ -45,3 +45,7 @@ It's important that your freshness jobs run frequently enough to snapshot data l
 | 1 hour | 30 mins |
 | 1 day | 12 hours |
 | 1 week | About daily |
+
+## Easy access
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-snapshotting-source-freshness.md]:
```suggestion
## Further reading
```

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-snapshotting-source-freshness.md [hunk 3]
```diff
@@ -45,3 +45,7 @@ It's important that your freshness jobs run frequently enough to snapshot data l
 | 1 hour | 30 mins |
 | 1 day | 12 hours |
 | 1 week | About daily |
+
+## Further reading
+
+For more on exposing links to the latest Documentation and Source Freshness reports to your team, see [Building and configuring artifacts](artifacts).
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-snapshotting-source-freshness.md]:
```suggestion
For more on exposing links to the latest documentation and sharing source freshness reports to your team, see [Building and configuring artifacts](artifacts).
```

**Pull request merged**